tools/bazel.rc

```diff
@@ -12,3 +15,37 @@ build --nojava_header_compilation # https://github.com/bazelbuild/bazel/issues/5
 
 # Ensure that we do not use an older Java version for the --javabase.
 build --javabase=@bazel_tools//tools/jdk:host_jdk
+
+build:remote --remote_instance_name=projects/kythe-repo/instances/default_instance --project_id=kythe-repo
```

Comment:
Please add a comment about where these settings came from (I'm assuming they have to be kept in sync with something upstream of us?)

Reply:
K.